app/controllers/application_controller.rb

```diff
@@ -1,2 +1,23 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  # before_action :authenticate_user!
```

Comment:
Please remove commented code from here as it can make the whole project unclean.

Reply:
Hello @SajjadAhmad14, the transactions views are not required for the functionality of the applications. I only use the category views. So, this does not apply to my application 👍